### PR TITLE
Add modules

### PR DESCRIPTION
**Pull request by sanjeevkopsai**

### CODE REVIEW

File: level2/load-balancer.tf [hunk 1]
```diff
@@ -1,67 +1,8 @@
...
-  load_balancer_arn = aws_lb.main.arn
-  port              = "80"
-  protocol          = "HTTP"
-
-  default_action {
-    type             = "forward"
-    target_group_arn = aws_lb_target_group.main.arn
-  }
-}
+}
```
Comment (geoffrey1330) [inline level2/load-balancer.tf]: could you please, add a newline at the end of file.

File: modules/vpc/main.tf [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  }
+}
+
+
+resource "aws_route_table_association" "private" {
+  count = length(var.private_cidr)
+
+  subnet_id      = aws_subnet.private[count.index].id
+  route_table_id = aws_route_table.private[count.index].id
+}
```
Comment (geoffrey1330) [inline modules/vpc/main.tf]: could you please, add a newline at the end of file.

File: modules/vpc/outputs.tf [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    value = aws_subnet.private[*].id
+}
+
+output "vpc_id" {
+    value = aws_vpc.main.id
+}
+
+output "vpc_cidr" {
+    value = var.vpc_cidr
+}
```
Comment (geoffrey1330) [inline modules/vpc/outputs.tf]: could you please, add a newline at the end of file.

**Review by geoffrey1330 — CHANGES_REQUESTED** [review]

File: modules/vpc/outputs.tf (hunk 3 above)
Comment (sanjeevkopsai) [reply modules/vpc/outputs.tf]: done

**Review by geoffrey1330 — APPROVED** [review]

**Pull request merged**